WORKFLOW.rst

```diff
@@ -0,0 +1,216 @@
+    renamed to the codename of that release, e.g., ``liberty-next``.
+
+Development Workflow
+--------------------
+
+#. After a bug or feature is prioritized and assigned, the developer pushes a
+   fix to a branch on |rpcos|. [#]_
+
+#. A PR is sent to target the |lsbnext| branch (if the bug applies to
+   |lsbnext|, otherwise it is sent to the most recent branch it affects.
+
+#. All features and bugs are merged to |lsbnext| as appropriate.
+
+#. Bugs meeting backport criteria [#]_ are backported to the previous branches
+   as appropriate.
+
+#. Backport PRs are added to the original issue by the developers who create
+   them for easier tracking.
+
+   .. code::
+
+       {{ Issue Description }}
+
+       ----
+
+       - [ ] Original PR: https://github.com/rcbops/rpc-openstack/pull/{{number}}
+
+       - [ ] Backport to kilo: https://github.com/rcbpos/...
+
+       - [ ] ...
+
+   Using markdown checklists in GitHub will give others a quick view into the
+   status of each pull request.
+
+#. Each time a PR is merged, the associated branch is deleted from |rpcos|.
+
+#. When all PRs are completed the issue is then closed.
+
+Release Workflow
+----------------
+
+.. note::
+
+    This proposed release workflow is meant to mimic some of OpenStack's
+    practices.
+
+#. Each time a large feature targetted for the next minor release is merged, a
+   Beta tag is created, e.g., ``r11.1.0b1``, ``r11.1.0b2``, etc.
+
+#. As we approach the end of features and bugs targetted for a minor release,
+   we create a Release Candidate tag, e.g., ``r11.1.0rc1``.
+
+#. Each tag that is created should be tested in depth by QE and/or Support for
+   acceptance and confirmation that the acceptance criteria were met. **This
+   is designed to create shorter feedback loops.**
+
+#. As bugs are identified in these tags, work is merged to the target branch.
+
+#. When a release is created, branches can be created from |lsbnext| if
+   desired by the stakeholders.
+
+Addendum
+^^^^^^^^
+
+If branches are desired for every major and minor release (but not patch),
+the branch naming scheme should change. An example would be to have
+branches named
+
+- ``r10.0``
+- ``r10.1``
+- ``r11.0``
+- ``r11.1``
+
+Bugs can then be backported to the ``r11.0`` and ``r11.1`` series as
+necessary.
+
+Alternative New Workflow
+========================
+
+The alternate workflow is similar to :ref:`the proposed workflow
+<proposed-workflow>`_ except that instead of backporting patches, patches are
+applied to the oldest applicable branch first, then second oldest, etc., until
+they are finally applied to |lsbnext|. This allows developers to use ``Closes
+#bugnumber`` in the commit message and for GitHub to automatically close the
+issue for us as a team.
+
+This is the workflow used by the `fabric`_ project.
+
+----
+
+.. Replacements
+
+.. |rpcos| replace:: ``rpc-openstack``
+.. |lsbnext| replace:: ``{{latest-stable-branch-name}}-next``
```

Comment:
latest-stable-branch-name is not as descriptive as I would like, as then we have to explain that a branch is stable when... and then have throw down qualifiers around work to try and get a next-cycle release together.